model/model.js

```diff
@@ -1,8 +1,29 @@
 const db = require("../db/connection.js");
 
-exports.selectTopics = () => {
+const selectTopics = () => {
     return db.query("SELECT * FROM topics;").then((selectTopicsQueryResult) => {
         const selectTopicsResponseObject = {topics: selectTopicsQueryResult.rows};
         return selectTopicsResponseObject;
     });
+};
+
+const selectArticles = () => {
+    const sqlQuery = `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, COUNT(comments.*) AS comment_count FROM articles LEFT JOIN comments ON comments.article_id = articles.article_id GROUP BY articles.article_id ORDER BY created_at DESC;`
+
+    return db.query(sqlQuery).then((queryResult) => {
+        // Convert values to correct formats
+        queryResult.rows.forEach((element) => {
```

Comment:
This can be achieved in the SQL statement using the `CAST` function. I would refactor to use this and remove the iteration.